VCFv4.3.tex

```diff
@@ -503,6 +507,10 @@ \subsubsection{Genotype fields}
   All phased genotypes that do not contain a PS subfield are assumed to belong to the same phased set.
   If the genotype in the GT field is unphased, the corresponding PS field is ignored.
   The recommended convention is to use the position of the first variant in the set as the PS identifier (although this is not required).
```

Comment:
This recommendation will also cause naming collisions since PSLs can span multiple chromsomes